week3/Week3/src/RemoveDuplicateElements.java

```diff
@@ -0,0 +1,36 @@
+public class RemoveDuplicateElements {
+
+    public static void RemoveDuplicateElements(int arr1[], int size)
```

Comment:
the funcation names should start with small character for the java name convention and best practice